looker/resource_looker_group.go

```diff
@@ -56,7 +57,7 @@ func resourceGroupCreate(ctx context.Context, d *schema.ResourceData, c interfac
 	}
 
 	if group.Id == nil {
-		return diag.Errorf("group %s has missing id", group.Name)
+		return diag.Errorf("group %s has missing id", name)
```

Comment:
I thought we stopped using this error with the name embedded since we're not checking if it's nil?

Reply:
Ah yes - this name is coming from the `ResourceData` rather than from the looker response, so I can be sure it's not nil because I cast it as a string here :)  https://github.com/resolutionlife/terraform-provider-looker/pull/14/files/7419b969e5c2259a21f2a4bab12b6dda50642411#diff-7def1142d24a4e599a44ac20b2fc0af2992962636e36c2d9e76ed7b4eaf19cbcR48